hbase-server/src/test/java/org/apache/hadoop/hbase/security/access/SecureTestUtil.java

```diff
@@ -71,7 +71,7 @@
 public class SecureTestUtil {
 
   private static final Logger LOG = LoggerFactory.getLogger(SecureTestUtil.class);
-  private static final int WAIT_TIME = 10000;
+  private static final int WAIT_TIME = 30000;
```

Comment:
Is this required?

Reply:
I enlarge the timeout because TestRSGroupsWithACL failed when grant permission timeout,  https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3024/1/testReport/
It is a small problem, I'll remove this change.